__tests__/app.test.js

```diff
@@ -122,3 +122,112 @@ describe("GET /api/articles", () => {
   });
 });
 
+describe("GET /api/articles/:article_id/comments", () => {
+  test("GET 200: array of comments for the given article_id of which each comment should have the certain properties and the comments should be served with the most recent comments first", () => {
+    return request(app)
+      .get("/api/articles/1/comments/")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeInstanceOf(Array);
+        expect(comments).toHaveLength(11);
+        const commentsCopy = [...comments];
+        const sortedComments = commentsCopy.sort((commentA, commentB) => {
+          return commentA.created_at - commentB.created_at
```

Comment:
This works, but consider using the jest-sorted package to this easily. 

https://www.npmjs.com/package/jest-sorted